client/scss/components/forms/_field-comment-control.scss

```diff
@@ -49,6 +49,14 @@
   }
 }
 
```

Comment:
Not sure if it's related but we have an override for these field types in `client/scss/components/forms/_input-text.scss`

https://github.com/wagtail/wagtail/blob/b9a5f9e333e6219141471f03d64aa16d927fc35b/client/scss/components/forms/_input-text.scss#L37-L44

Reply:
Yes, this is related and this is why the width of the `datefield` input is smaller than the width of the other input fields as seen below;

![wagcommentissuereproduced](https://user-images.githubusercontent.com/62052443/199225858-84a5f1c9-d545-4243-aedc-0d1e8323b46c.PNG)

Becuase of the adjustment of width here

 // Make non-text field types with custom widgets have a smaller width. 
 .w-field--date_field, 
 .w-field--date_time_field, 
 .w-field--time_field { 
   input { 
     width: auto; 
   } 
 } 

The comment button and `datefield` input has a noticeably large space in between because of the below code (client/scss/components/forms/_field-comment-control.scss):

.w-field__comment-button {
  $root: &;

  @include transition(opacity 0.2s ease);
  position: absolute;